<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpPoolAnalyserDialog</class>
 <widget class="QDialog" name="IpPoolAnalyserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>351</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>IP Pool Analyser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../image.qrc">
    <normaloff>:/Image/analyze.png</normaloff>:/Image/analyze.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="osCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 0, 127);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../image.qrc">:/Image/analyze.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="cmdLbl_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>Command :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="cmdLbl">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 0, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 53, 102);</string>
       </property>
       <property name="text">
        <string>Select Captured File</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../image.qrc">
         <normaloff>:/Image/brows.png</normaloff>:/Image/brows.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 0, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>Pool Count :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="poolContLbl">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>5000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>Total IP Count :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalIpLbl">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>5000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>Public IP Count :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="publicIpLbl">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>5000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 127);</string>
          </property>
          <property name="text">
           <string>Private IP Count :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="privateIpLbl">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>5000</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 0, 127);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>osCB</tabstop>
  <tabstop>btn</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../image.qrc"/>
 </resources>
 <connections/>
</ui>
